pagopa/io-app
# chore: [LLK-57] Fix unsupported device banner visibility

## Short description
This PR fixes the concurrency critical path between the already opted in user banner management and the backend status retrieval.

## List of changes proposed in this pull request.
- ts/features/lollipop/saga/index.ts: Sychronizes with the backend status API call.
- ts/sagas/startup.ts: wrap the saga with `call`.

## How to test
- Verify that the banner is not displayed when Lollipop is disabled, regardless of whether a public key is present or not.
- Confirm that the banner is not displayed when Lollipop is enabled and a public key is present.
- Ensure that the banner is displayed when Lollipop is enabled and a public key is not present.
- Ensure that by tapping on the banner you navigate to the link proposed [here](https://pagopa.atlassian.net/browse/LLK-52).

Make all the above tests either with an old app where you are already logged in and then update to the new one, and with the new one and a brand new login on both Android and iOS devices.
